pkg/tid/cli/command/entry/list.go

```diff
@@ -85,6 +85,7 @@ func ListCommand(factory util.Factory) *console.Command {
 
 		if len(entries) == 0 {
 			output.Println("list: No entries within the given time period")
+			output.SetExitCode(1)
```

Comment:
For these, I think we've already discussed, you can instead return an error to have it be a one-liner that sets the exit code too (though the addition of SetExitCode is still very useful).